--- lib/src/asn1/rrc/dl_dcch_msg.cc	2023-02-05 05:09:31.177232619 +0100
+++ /root/srsran_to_umts/lib/src/asn1/rrc/dl_dcch_msg.cc	2023-02-05 05:26:44.959923465 +0100
@@ -8533,7 +8533,8 @@
 
 void rrc_conn_release_v920_ies_s::cell_info_list_r9_c_::destroy_()
 {
-  switch (type_) {
+  types type_h4ck3d = types::geran_r9;
+  switch (type_h4ck3d) {
     case types::geran_r9:
       c.destroy<cell_info_list_geran_r9_l>();
       break;
@@ -8554,7 +8555,8 @@
 {
   destroy_();
   type_ = e;
-  switch (type_) {
+  types type_h4ck3d = types::geran_r9;
+  switch (type_h4ck3d) {
     case types::geran_r9:
       c.init<cell_info_list_geran_r9_l>();
       break;
@@ -8577,7 +8579,8 @@
     const rrc_conn_release_v920_ies_s::cell_info_list_r9_c_& other)
 {
   type_ = other.type();
-  switch (type_) {
+  types type_h4ck3d = types::geran_r9;
+  switch (type_h4ck3d) {
     case types::geran_r9:
       c.init(other.c.get<cell_info_list_geran_r9_l>());
       break;
@@ -8603,7 +8606,8 @@
     return *this;
   }
   set(other.type());
-  switch (type_) {
+  types type_h4ck3d = types::geran_r9;
+  switch (type_h4ck3d) {
     case types::geran_r9:
       c.set(other.c.get<cell_info_list_geran_r9_l>());
       break;
